core/src/main/java/com/linecorp/armeria/client/endpoint/dns/DnsServiceEndpointGroup.java

```diff
@@ -66,11 +66,12 @@ public static DnsServiceEndpointGroupBuilder builder(String hostname) {
     }
 
     DnsServiceEndpointGroup(EndpointSelectionStrategy selectionStrategy, EventLoop eventLoop, Backoff backoff,
-                            int minTtl, int maxTtl, String hostname, DefaultDnsResolver resolver) {
+                            int minTtl, int maxTtl, String hostname, DefaultDnsResolver resolver,
+                            boolean allowEmptyEndpoints) {
```

Comment:
How about reordering the constructor parameters, so that all `DynamicEndpointGroup` constructor parameters come before the others? i.e.

```
DnsServiceEndpointGroup(EndpointSelectionStrategy, boolean, ...)
```